api/net/Areas/Editor/Controllers/ContentController.cs

```diff
@@ -504,7 +504,10 @@ public async Task<IActionResult> StreamAsync([FromQuery] string path)
         using var memoryStream = new MemoryStream();
         await fileStreamResult.FileStream.CopyToAsync(memoryStream);
         var result = Convert.ToBase64String(memoryStream.ToArray());
-
+        // TODO: Out of Memory issues with this implementation.
```

Comment:
@CelloLiu Your implementation is constantly throwing OutOfMemoryExceptions.  This will not work for us.  I've created a story to resolve this.